tiledb/sm/storage_manager/open_array.h

```diff
@@ -88,6 +88,12 @@ class OpenArray {
   /** Returns the array schema. */
   ArraySchema* array_schema() const;
 
+  /** Returns array schemas map. */
+  std::unordered_map<std::string, tdb_shared_ptr<ArraySchema>>& array_schemas();
+
+  /** Returns the array schema given a array schema name. */
+  tdb_shared_ptr<ArraySchema> get_array_schema(const std::string& schema_name);
```

Comment:
We typically use `get_*` when we get the object by passing a pointer in the argument. I suggest we simply overload the `array_schema` function.

Reply:
changed to pass a pointer in the argument and return the Status